pipeline_steps/common.groovy

```diff
@@ -3,7 +3,8 @@ import groovy.json.JsonOutput
 
 // Install ansible on a jenkins slave
 def install_ansible(){
-  sh """#!/bin/bash
+  sh """#!/bin/bash -xe
+    cd ${env.WORKSPACE}
```

Comment:
This implies the location of the venv will change, does that matter? Will it have any impact elsewhere, e.g. https://github.com/rcbops/rpc-gating/blob/master/pipeline_steps/common.groovy#L345?